src/api/app/views/webui2/webui/users/subscriptions/index.js.erb

```diff
@@ -0,0 +1 @@
+$('#subscriptions-form').replaceWith('<%= escape_javascript(render(partial: 'webui/subscriptions/subscriptions_form')) %>');
```

Comment:
We don't use ERB for new files.

Reply:
but?

Reply:
We use HAML now.

Reply:
but this is not html, but javascript

Reply:
I think this should be inline in the subscriptions/index view with `javascript: [..]`. Then this file should be deleted. Sorry for the confusion @krauselukas, I should have been clearer from the start.

Reply:
This view is probably rendered when the `update` action is called. If you move it to the html views, you might have to update how we handle the ajax requests (and their response).

And changing that might also affect the old views. I would postpone this until after the migration.

Reply:
@dmarcoux would you be fine with postponing it?

Reply:
Yes, no worries.